src/components/application/StatsResults.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+
+export default function({ body }) {
+
+    const [ user, setUser ] = useState("");
+
+    function ForUser({ user }) {
+
+        const loginEvents = body;
+        console.log(loginEvents);
+        return <article>
+            <table>
+                <tbody>
+                    <tr>
+                        <th>DatePart</th>
+                        <th>Count</th>
+                    </tr>
+                    {loginEvents.map(result => (<tr>
+                        <td>{result.DatePart}</td>
+                        <td>{result.Count}</td>
+                    </tr>))}
+                </tbody>
+            </table>
+        </article>;
+    }
+
+    return <main>
```

Comment:
We should only have 1 `main` element (and there is one higher up).

I also think this return should have parenthesis. Helps with consistent indentation;
```
return (
  <div>
    <h1>Thing</h1>
  </div>
);
```

Reply:
Updated